apps/e2e/a11y-storybook-e2e/src/e2e/skip-link.component.cy.ts

```diff
@@ -1,5 +1,6 @@
 import { E2eVariations } from '@skyux-sdk/e2e-schematics';
 
+/* spell-checker:ignore skiplinkcomponent, skiplink */
```

Comment:
In a perfect world, our source code wouldn't have any reference to a spell checker. Could we instead ignore these words in a root-level `.cspell.json` file?

Reply:
Throughout this process there are a lot of these. Lets flex on this. For these we could - but others I don't know that I want to - I'm not sure I fully follow why we would not do these in this way - as it is the same as lint ignores.